README.md

```diff
@@ -62,6 +71,47 @@ To run GitGlance locally, follow these steps:
 ## Contributing
 Contributions are welcome! If you encounter any issues or have suggestions for improvements, please feel free to open an issue or submit a pull request.
 
+
+## How to Contribute
+We welcome contributions from the community to enhance GitGlance further. Here's how you can contribute:
+
+# 1. Fork the Repository
+Fork the GitGlance repository by clicking the "Fork" button at the top right corner of this page. This will create a copy of the repository in your GitHub account.
+
+# 2. Clone the Repository
+Clone your forked repository to your local machine using GitHub Desktop or by running the following command in your terminal:
+
+```bash
+git clone https://github.com/your-username/gitglance.git
+
```

Comment:
I think there is something wrong. Please fix it.

![Screenshot at 2024-05-12 15-46-03](https://github.com/devXprite/gitglance/assets/80192140/18c3d75f-b0ee-41e4-9944-999bb279ba24)